includes/wikia/tests/core/WikiaBaseTest.class.php

```diff
@@ -348,7 +317,11 @@ protected function getConstructorMock( $className ) {
 	 */
 	protected function getGlobalFunctionMock( $functionName ) {
 		list($namespace,$baseName) = WikiaMockProxy::parseGlobalFunctionName( $functionName );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$namespace" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)